docs/security-considerations.rst

```diff
@@ -173,8 +172,8 @@ Never use tx.origin for authorization. Let's say you have a wallet contract like
         }
 
         function transfer(address dest, uint amount) {
-            if (tx.origin != owner) { throw; }
-            if (!dest.call.value(amount)()) throw;
+            require(tx.origin == owner);
+            require(dest.call.value(amount)());
```

Comment:
@axic 125 - 130 will need to be updated to reflect this change, I am not confident that I understand deeply enough to replace this paragraph; would you mind taking a look and if we still want to switch to `transfer()` let me know what you'd like me to replace that paragraph with.